package/share-root.sh

```diff
@@ -4,5 +4,3 @@ set -x
 trap 'exit 0' SIGTERM
 ID=$(grep :devices: /proc/self/cgroup | head -n1 | awk -F/ '{print $NF}' | sed -e 's/docker-\(.*\)\.scope/\1/')
 bash -c "$1"
-
-docker kill --signal=SIGTERM $ID
```

Comment:
If we are sure that the share-mnt process exit with 0 the I am good with the change.